romancal/resample/resample.py

```diff
@@ -86,13 +86,15 @@ def __init__(
             )
 
         self.input_models = input_models
+        # add sky variance array
+        resample_utils.add_var_sky_array(self.input_models)
```

Comment:
This will always re-compute `var_sky` (even if the input contains a `var_sky`). If that's the goal (always have resample compute `var_sky`) what about something like:
- if `weight` is `ivsky` compute `var_sky` for each model
- when resampling `var_sky` compute it (if it wasn't computed before)

That way for non`ivsky` weighting resample won't need to make a separate pass through all the models here (in `add_var_sky_array`).